sdks/java/io/hadoop/input-format/src/main/java/org/apache/beam/sdk/io/hadoop/inputformat/HadoopInputFormatIO.java

```diff
@@ -389,6 +394,7 @@ public void populateDisplayData(DisplayData.Builder builder) {
     private long boundedSourceEstimatedSize = 0;
```

Comment:
looking at the code some more - in https://github.com/diptikul/incubator-beam/blob/HIFIO-CS-ES/sdks/java/io/hadoop/input-format/src/test/java/org/apache/beam/sdk/io/hadoop/inputformat/HadoopInputFormatIOTest.java#L164 - we should hard code expected values there.